docs/changelog.rst

```diff
@@ -2,6 +2,15 @@
 CHANGELOG
 =========
 
+**🚀 Nouveautés POPAmphibien  POPReptile**
+
+* Ajout des sous modules POPAmphibien et POPReptile
+* Modification de la couleur de la partie superieure des detailles en bleu
+* Dans la partie map, joindre les sites par des lignes pour former une aire et calculer sa superficie (si le nombre des points est suerieur à 2)
+* Ajout d'un bouton pour l'export PDF de la partie aire
+* Ajout d'un bouton pour le telechargement des données standard et d'analyse pour chaque sous module en format CSV via un modale
+* Correction de "Point d'observations" par "Points d'observation"
+* alter table gn_monitoring.t_base_sites alter column id_nomenclature_type_site drop not null; 
```

Comment:
C'est une contrainte qui vient du coeur de geonature

plutot que de l'enlever le NOT NULL, on peut, pour pour la définir `id_nomenclature_type_site` comme un champs caché avec une valeur par défaut

au besoin on peut rajouter un element de nomenclature pour le type `TYPE_SITE` pour avoir un type de site qui va correspondre aux besoins du module

Reply:
ok pour une valeur par défaut